spec/models/ontology_version_spec.rb

```diff
@@ -34,6 +34,26 @@
       stub_hets_for(hets_out_file('pizza'))
     end
 
+    context 'in subdirectory' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.